README.md

```diff
@@ -44,11 +46,15 @@ Set the endpoint to be:
 
 Hubstats needs to know what repos for it to watch. You can set it to watch either an entire organization or a list of specific repos in octokit.yml.
 
+### The `test` Directory
+
+`test` directory exists because since hubstats is a plugin that's designed to work inside of an application, in order to view it locally and to test what the plugin would look like, we need to install the plugin into a greater application to run. This is the purpose of the `test` directory. `test/dummy` contains a plugin that automatically syncs up with the other code. When in the development process, one just needs to run `rails s` from the command line (while in `test/dummy`) to serve the plugin.
```

Comment:
Consider making the header more generic to cover testing.  Describe that automated tests are written in RSpec and the test folder contains a dummy rails application for manually testing the UI.

Also, "because since" is confusing.  Instead of "greater application", call it a Rails application.  Consider linking to the Rails plugin guide to do some of the explaining for you.